core/clingutils/CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ include_directories(${LLVM_INCLUDE_DIRS} ${CLING_INCLUDE_DIRS})
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/res ${CMAKE_CURRENT_SOURCE_DIR}/../foundation/res)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CLING_CXXFLAGS}")
 
+# This is to avoid warnings coming from GCC 7 in llvm/src/include/llvm/ADT/DenseMap.h:1010
```

Comment:
If it is a false positive, could we submit a gcc bug report and exclude the diagnostics only for this file. Something like:
```
# FIXME: This works around gcc bug id: N
set_source_files_properties(src/rootcling_impl.cxx   PROPERTIES COMPILE_FLAGS "-Wno-maybe-uninitialized")
```

That would help when reducing the suppression.